parentchain/src/tests.rs

```diff
@@ -0,0 +1,47 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+use crate::mock::*;
+use frame_support::{assert_err, assert_ok};
+use sp_core::H256;
+use sp_keyring::AccountKeyring;
+use sp_runtime::DispatchError::BadOrigin;
+
+#[test]
+fn verify_storage_works() {
+	new_test_ext().execute_with(|| {
+		assert_ok!(Parentchain::insert_block(
+			Origin::root(),
+			3,
+			H256::from_low_u64_be(42),
```

Comment:
This might be a personal preference, but I think in case of using the same variable twice in testing it makes sense to allocate it. It will make things easier to adapt in case the tests need to be changed. For example parent_hash needs to be removed from insert block, it'd be clear which one is the parent_hash in the test fn and the compiler will throw errors in case something is forgotten to be deleted.. in this case the `assert_eq!` from below.